benchmark/benchmarks/krausest/lib/utils/data.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
```

Comment:
The biggest change across the entire PR is the addition of `eslint-disable` directives to account
for the more consistent application of eslint rules in this PR.